lib/infra/infra-stack.ts

```diff
@@ -824,6 +825,19 @@ export class InfraStack extends Stack {
       currentWorkDir = '/mnt/data';
     }
 
+    // Add 2G swap for t3.medium instances
+    if (instanceType === 't3.medium') {
+      cfnInitConfig.push(InitCommand.shellCommand('set -ex; '
+        + 'sudo dd if=/dev/zero of=/swapfile bs=1M count=2048; '
+        + 'sudo chmod 600 /swapfile; '
+        + 'sudo mkswap /swapfile; '
+        + 'sudo swapon /swapfile; '
+        + 'echo "/swapfile swap swap defaults 0 0" | sudo tee -a /etc/fstab',
+      {
+        ignoreErrors: false,
+      }));
+    }
```

Comment:
How about add `enable2GSwap = true` for any instance type rather than adding customization?